### PR TITLE
[export wrapper] support simple image name

### PR DESCRIPTION
**Pull request by dntczdx**
[export wrapper] support simple image name.
Before: only support format like "global/images/anImage"
Now: support "anImage" as well, which will be extended to "global/images/anImage" automatically

### CODE REVIEW

File: cli_tools/gce_vm_image_export/exporter/exporter.go [hunk 1]
```diff
@@ -101,6 +102,14 @@ func buildDaisyVars(destinationURI string, sourceImage string, format string, ne
 	return varMap
 }
 
+func getFullImagePath(imageName string) string {
+	if !strings.Contains(imageName, "/") {
+		// Extend simple image name to full image name
+		return fmt.Sprintf("global/images/%v", imageName)
```
Comment (zoran15) [inline cli_tools/gce_vm_image_export/exporter/exporter.go]: Is this going to work with project specific images?

Comment (dntczdx) [reply cli_tools/gce_vm_image_export/exporter/exporter.go]: It's "current project image". User can specify any project as current project by --project=aProject.
If the image is from a different project from running project, user has to specify full image path - "projects/another-project/global/images/animage"

**Comment by dntczdx** [comment]
I have 

> Can you please add e2e test for export wrapper to make sure this actually works as intended?

I have added unit test for it. It's too expensive to run a e2e for such a minor change in long term in my opinion. I have ran the wrapper manually to confirm it works.

**Comment by zoran15** [comment]
> I have
> 
> > Can you please add e2e test for export wrapper to make sure this actually works as intended?
> 
> I have added unit test for it. It's too expensive to run a e2e for such a minor change in long term in my opinion. I have ran the wrapper manually to confirm it works.

What about modifying one of the existing tests to use short name?

**Comment by dntczdx** [comment]
> > I have
> > > Can you please add e2e test for export wrapper to make sure this actually works as intended?
> > 
> > 
> > I have added unit test for it. It's too expensive to run a e2e for such a minor change in long term in my opinion. I have ran the wrapper manually to confirm it works.
> 
> What about modifying one of the existing tests to use short name?

Good idea, I used it in rich_param test due that it covers all unusual params

**Comment by dntczdx** [comment]
Ping @zoran15

**Pull request merged**